pages/opportunities/index.tsx

```diff
@@ -12,6 +12,7 @@ import { Route } from "lib/routing";
 import { useState } from "react";
 import { siteData } from "lib/site-data";
 import strings from "content/strings.json";
+import Select from "../../components/select";
```

Comment:
Hnida: Je možné používat relativní cestu vůči kořenu repa, přijde mně to čitelnější: `components/select`. Můžem nahradit?

Reply:
Dobrej point, sjednotil jsem to teda se zbytkem :)